doc/elements-getting-started.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Getting Started
+order: 2
+layout: page
+---
+
+# Getting Started
+
+This page will guide you through the installation of Vaadin Core Elements bundle and help you get started with your first project using these elements.
+
+## Installation
+
+There are three ways to use Vaadin Core Elements in your project: Bower, CDN and a ZIP archive. The only difference between these options is the URL you will use to import the necessary files into your HTML page.
+
+### 1. Create a new folder
+
+Start by creating a new folder for your project and change into the newly created folder.
+
+      $ mkdir my-project
+      $ cd my-project
+
+### 2. Install Vaadin Core Elements
+
+#### Bower
+
+Recommended way to manage your front-end dependencies is using [Bower](http://bower.io). Follow the [Bower installation instructions](http://bower.io/#install-bower), then run the following command inside your project folder to install the most recent stable release.
+
+
+     $ bower install --save vaadin-core-elements
```

Comment:
Any good reason for adding `--save` ? 
I would remove it, since it's not mandatory for installing and using the component.